packages/loot-core/src/server/accounts/transaction-rules.ts

```diff
@@ -16,8 +16,7 @@ import * as db from '../db';
 import { getMappings } from '../db/mappings';
 import { RuleError } from '../errors';
 import { requiredFields, toDateRepr } from '../models';
-import { batchMessages } from '../sync';
-import { addSyncListener } from '../sync/index';
+import { batchMessages, addSyncListener } from '../sync';
```

Comment:
Do you know if there’s an ESLint rule for preventing this duplication from happening?

Reply:
Looks like, https://eslint.org/docs/latest/rules/no-duplicate-imports

I'll have a look at adding this after my current round of eslint changes?